x/shield/types/msg.go

```diff
@@ -493,3 +493,85 @@ func (msg MsgWithdrawReimbursement) GetSignBytes() []byte {
 func (msg MsgWithdrawReimbursement) ValidateBasic() error {
 	return nil
 }
+
+// TODO: eliminate this msg type
```

Comment:
Why eliminate?

Reply:
because we ideally should merge this into PurchaseStaking message type. They are essentially the same.

Reply:
Let's not reuse the same message type for different txs